andela/ah-titans-frontend
# #160244844 Setup code coverage script

#### What does this PR do?
Sets up code coverage

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ch-setup-coverage-script-160244844
```
```sh
$ cd ah-titans-frontend
```

>Install the dependencies using the command below.

```sh
$ npm install
```
>Run the command below on your command line.
```sh
$ npm run coverage
```
>You should be able to see the test coverage per folder  as shown on the screenshot below.

![image](https://user-images.githubusercontent.com/29063646/45031813-a5cbaf00-b058-11e8-9b2b-58c68d610e90.png)


#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160244844](https://www.pivotaltracker.com/story/show/160244844)
